samples/snippets/src/test/java/com/example/dialogflow/CreateConversationTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.dialogflow;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.dialogflow.v2.Conversation;
+import com.google.cloud.dialogflow.v2.Conversation.LifecycleState;
+import com.google.cloud.dialogflow.v2.ConversationProfile;
+import com.google.cloud.dialogflow.v2.ConversationProfileName;
+import com.google.cloud.dialogflow.v2.ConversationProfilesClient;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class CreateConversationTest {
+  private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static final String CONVERSATION_PROFILE_DISPLAY_NAME = UUID.randomUUID().toString();
+  private static final String LOCATION = "global";
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+  private String conversationProfileId;
+
+  private static void requireEnvVar(String varName) {
+    assertNotNull(System.getenv(varName));
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireEnvVar("GOOGLE_APPLICATION_CREDENTIALS");
```

Comment:
Maybe not required.

Reply:
It is actually required for running tests.
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.kokoro/tests/run_tests.sh#L58